src/Microsoft.ComponentDetection.Contracts/Mappers/CycloneDx.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using CycloneDX.Json;
+using CycloneDX.Models;
+using Microsoft.ComponentDetection.Contracts.BcdeModels;
+
+namespace Microsoft.ComponentDetection.Contracts.Mappers
+{
+    public static class CycloneDx
+    {
+        public static string ToCycloneDx(this ScanResult scanResult)
+        {
+            return Serializer.Serialize(
+                new Bom
+                {
+                    Metadata = new Metadata
+                    {
+                        Tools = new List<Tool>
+                        {
+                            new Tool
+                            {
+                                Vendor = "Microsoft",
+                                Name = "Component Detection",
+                            },
```

Comment:
It would be great to also include the tool version, as well as an external reference to the project on GitHub.